src/assets/scripts/client/scope/RadarTargetModel.js

```diff
@@ -303,6 +306,72 @@ export default class RadarTargetModel {
         return [true, 'AMEND ALTITUDE'];
     }
 
+    /**
+     * Generate a string to be used for the first row of a datablock
+     *
+     * @for RadarTargetModel
+     * @method buildDataBlockRowOne
+     * @returns {string}
+     */
+    buildDataBlockRowOne() {
+        let dataBlockRowOne = `${this.aircraftModel.callsign}`;
+
+        if (this.aircraftModel.model.weightclass === 'H') {
+            dataBlockRowOne = `${this.aircraftModel.callsign}  ${this.aircraftModel.model.weightclass}`;
+        }
+
+        return dataBlockRowOne;
+    }
+
+    /**
+     * Generate a string to be used for the second row of a datablock
+     *
+     * @for RadarTargetModel
+     * @method buildDataBlockRowTwo
+     * @returns {string}
+     */
+    buildDataBlockRowTwo() {
+        const aircraftAltitude = round(this.aircraftModel.altitude * 0.01);
+        const aircraftSpeed = round(this.aircraftModel.groundSpeed * 0.1);
+
+        return `${leftPad(aircraftAltitude, 3)} ${leftPad(aircraftSpeed, 2)}`;
+    }
+
+    /**
+     * Generate a string to be used for the second row of a datablock
+     * when the timeshare section is active
+     *
+     * @for RadarTargetModel
+     * @method buildDataBlockRowTwoTimeshare
+     * @returns {string}
+     */
+    buildDataBlockRowTwoTimeshare() {
+        return `${this.aircraftModel.timeShareDestination} ${this.aircraftModel.model.icao.toUpperCase()}`;
```

Comment:
The first element here (the "scratchpad") must be trimmed or padded to be 4 characters exactly to maintain the appropriate spacing.
```
|---|---|
R2   A320
KSEA B738
PAE  B712
|---|---|
```